### DISCUSSION TITLE
what makes a TF excluded from scenicplus?

### DISCUSSION THREAD

**Discussion by janursa** [aertslab/scenicplus#245] [General]
Hi Seppe,
Thank you for such a nice work. I went through your tutorial and got the expected results. When i check the TFs in eRegulons, i see that there is around 100 left from the initial ~1000 included in the study (i see ~1200 in TF2G_adj). I wonder, what makes a TF excluded from the final list? i assume one reason is that you only included HVGs (~5000 genes), which might reduce TF size. Also, you use aucell which also filters out sources by default. 
I found [this discussion](https://github.com/aertslab/scenicplus/discussions/178) to be also related to my question.

**Comment by SeppeDeWinter** [comment]
Hi @janursa 

Thank you.

Most TFs will be "lost"/"filtered out" by the motif enrichment step. We only keep TFs if they have a motif enriched. 

Furthermore, this set of TFs is filtered down based on the TF-to-gene and region-to-gene adjacencies. The target genes of each TF is defined as the set of genes linked to regions enriched for the TF's motif(s) that are also located in the top of the TF-to-gene ranking (based on the TF-to-gene importance score). If less than 10 target genes are found (default settings) the TF will be filtered out.

I would not recommend to only use HVGs!

AUCell should not be filtering out any TF.


I hope this helps.

All the best,

Seppe
